content/pages/playbook/product-methodology.html

```diff
@@ -29,18 +29,17 @@
       </div>
       <div class="small-12 usa-width-three-fourths medium-9 columns">
         <ol class="va-list-num--discover">
```

Comment:
If possible, it might be best to leave the Playbook styles unchanged for now since we're planning to move a lot of it away from static pages and into a GH wiki. Otherwise this is happening:

![image](https://cloud.githubusercontent.com/assets/4106356/25509251/676063e0-2b6c-11e7-986c-8e72b0cd9927.png)